src/testTrackConfig.js

```diff
@@ -10,6 +11,10 @@ var TestTrackConfig = (function() { // jshint ignore:line
         };
 
     return {
+        _clear: function() {
```

Comment:
At some point it might make sense to refactor into some sort of configfactory/builder/something. But as far as an immediate fix this seems ok as long we don't continue to lean on it for more and more things.